qemu/tests/cfg/hotplug_mem_stress_ng.cfg

```diff
@@ -0,0 +1,27 @@
+- hotplug_mem_stress_ng:
+    type = hotplug_mem_stress_ng
+    mem_fixed = 4096
+    slots_mem = 4
+    maxmem_mem = 32G
+    pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
+    no Host_RHEL.6
+    no RHEL.5 RHEL.6 RHEL.7
+    no Windows
+    # Notes:
+    #    The threshold on ppc was confirmed with developer
+    #    since crashkernel size is much bigger than x86
+    ppc64,ppc64le:
+        threshold = 0.15
+    guest_numa_nodes = "node0"
+    mem_devs = "mem0"
+    use_mem_mem0 = "no"
+    size_mem_mem0 = 4096M
+    numa_memdev_node0 = mem-mem0
+    backend_mem = memory-backend-ram
+    target_mems = "plug1"
+    size_mem_plug1 = 1G
+    stress_ng_dir = "/tmp/stress-ng"
+    get_stress_ng = "[ -d ${stress_ng_dir} ] && rm -rf ${stress_ng_dir}; git clone --depth=1 git://kernel.ubuntu.com/cking/stress-ng.git ${stress_ng_dir}"
+    compile_stress_ng = "cd ${stress_ng_dir} && make -j4"
+    run_stress_ng = "cd ${stress_ng_dir} && ./stress-ng %s"
+    stress_ng_args = "--page-in -r 4 -t 60s;--malloc 4 -t 60s;--mmap 4 -t 60s;--cpu 5 --vm 9 --vm-bytes 40% --iomix 6 --iomix-bytes 40% --memcpy 2 --cache-prefetch -t 1200s"
```

Comment:
`get_stress_ng` and `stress_ng_args` looks too long, could you split it ?

Reply:
Do we have 80 chars limitation for cfg too? If yes, I have to split git clone cmd.

Reply:
> Do we have 80 chars limitation for cfg too? If yes, I have to split git clone cmd.

IMO, `git clone --depth=1 git://kernel.ubuntu.com/cking/stress-ng.git ${stress_ng_dir}"` is acceptable.
To make more readable, 80 limitation is not must ~  we can balance between the two.

Reply:
Okay, updated.